Yoast/wordpress-seo
# Move license manager into codebase

⚠️ This isn't ready yet ⚠️ 

## Left to do

- [ ] Fix translator comments for strings.
- [ ] Fix code documentation.
- [ ] Fix CodeClimate smells.
- [ ] Fix CS errors.
- [ ] Add a PHP 5.2 fallback for `class_alias`.
- [ ] Make the API request do a `POST` instead of a `GET`, which unfortunately doesn't work "easily".

## Summary

This PR can be summarized in the following changelog entry:

* Removed external dependency on license manager.

## Relevant technical choices:

* Addressed all to-do's in #9248 

## Test instructions

This PR can be tested by following these steps:

* Check activating and deactivating plugins in MyYoast and see the impact in the SEO -> Premium menu item.

## Quality assurance


Fixes #9248
